### PR TITLE
fix conference schedule typo

### PR DESCRIPTION
**Pull request by dustinloflandsmith**
updated conference schedule page to avoid unnecessary time travel. If the rst syntax is wrong, please hard code Monday, September 16 and Tuesday, September 17 instead 😄

### CODE REVIEW

**Comment by plaindocs** [comment]
Thanks for noticing and fixing @dustinloflandsmith ! 

You can see why the day 4 variable doesn't work here https://github.com/writethedocs/www/blob/master/docs/_data/config-prague-2019.yaml#L65

File: docs/conf/prague/2019/schedule.rst [hunk 1]
```diff
@@ -134,7 +134,7 @@ There will be light snacks and drinks available on the conference while our tab
 
 Further details will be announced later.
 
-Tuesday, September 11
+{{date.day_four.dotw}}, {{date.day_four.date}}
```
Comment (plaindocs) [inline docs/conf/prague/2019/schedule.rst]:
```suggestion
.. TODO add this variable? 

Tuesday, September 17
```

**Comment by dustinloflandsmith** [comment]
Makes total sense; thanks for the RST lesson, @plaindocs 😉 I've gone ahead and put the dates in manually for now.

**Comment by plaindocs** [comment]
Thanks!

**Review by plaindocs — APPROVED** [review]

**Pull request merged**